src/MongoDB.Bson/IO/JsonWriter.cs

```diff
@@ -139,6 +139,11 @@ public override void WriteBinaryData(BsonBinaryData binaryData)
                     _textWriter.Write("{{ \"$binary\" : \"{0}\", \"$type\" : \"{1}\" }}", Convert.ToBase64String(bytes), ((int)subType).ToString("x2"));
```

Comment:
Should we add the same change for WriteDecimal128 here?
cc: @rstam

Reply:
I don't know what change you are referring to.

Reply:
I mean mentioning new json output modes here: https://github.com/mongodb/mongo-csharp-driver/blob/master/src/MongoDB.Bson/IO/JsonWriter.cs#L238

Reply:
Got it.

In `WriteDecimal128` the `default` case is already canonical extended JSON.

Are you just proposing this:

    case JsonOutputMode.CanonicalExtendedJson:
    case JsonOutputMode.RelaxedExtendedJson:
    default:

Reply:
yep